common/lib/xmodule/xmodule/course_module.py

```diff
@@ -774,6 +774,30 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    learning_info = List(
+        display_name=_("Course Learning Information"),
+        help=_("Specify what student can learn from the course."),
+        default=[''],
+        scope=Scope.settings
+    )
+
+    instructor_info = Dict(
+        display_name=_("Course Instructor"),
+        help=_("Enter the details for Course Instructor"),
+        default={
```

Comment:
Yes, Initially, I am rendering the instructor info section with an empty fields. If it is not the requirement we can update it. Actually, other dict and list fields here are also using some default values. I followed the same approach.